Frameworks/UIKit/UISegment.mm

```diff
@@ -220,7 +224,7 @@ - (id)drawRect:(CGRect)inRect {
     if (isOSTarget(@"7.0")) {
         if (_tintColor != nil) {
             if (_selected == 1) {
-                CGContextSetFillColorWithColor(UIGraphicsGetCurrentContext(), (CGColorRef)_tintColor);
+                CGContextSetFillColorWithColor(UIGraphicsGetCurrentContext(), (CGColorRef)_tintColor.get());
```

Comment:
>_tintColor [](start = 90, length = 10)

This scares me a little. I know our CGColor _is_ a UIColor, and this code isn't yours... but would you mind adding a call to `-[UIColor CGColor]` for these?